themes/admin/Sea_Green/css/sea_green.css

```
@@ -16520,6 +16520,11 @@ body {
   flex-direction: column;
 }
 
+.topbar-back-link:focus {
+  outline: 2px solid currentColor !important;
```

Stylelint (17.5.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)